spec/features/brexit_checker/question_results_spec.rb

```diff
@@ -65,6 +71,114 @@ def when_i_visit_the_brexit_checker_flow
     visit brexit_checker_questions_path
   end
 
+  def and_i_should_see_citizen_actions_are_grouped
```

Comment:
figured i should test that results actually appear in groups, only looking at the headings, seemed to be in line with the other tests in here